<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>511</width>
      <height>361</height>
     </rect>
    </property>
    <widget class="QWidget" name="homeScreen">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>481</width>
        <height>341</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="lblInsulin">
           <property name="text">
            <string>Insulin: 300U</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="lblIOB">
           <property name="text">
            <string>IOB: 0.0U</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="lblBattery">
           <property name="text">
            <string>Battery: 100%</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="btnBolus">
         <property name="text">
          <string>Bolus</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btnProfiles">
           <property name="text">
            <string>Profiles</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnHistory">
           <property name="text">
            <string>History</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="bolusScreen">
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>461</width>
        <height>231</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="1">
        <widget class="QLineEdit" name="txtCarbs">
         <property name="text">
          <string>Carbs (Grams):</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="txtGlucose">
         <property name="text">
          <string>Glucose (mg/dL): </string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="lblBolusResult">
         <property name="text">
          <string>Suggested Bolus: 0.0U</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btnCalculate">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>250</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Calculate</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDeliver">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>250</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Deliver</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page"/>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
